installer/functions/task-router/deploy-task-router.ts

```diff
@@ -0,0 +1,124 @@
+// Imports global types
+import '@twilio-labs/serverless-runtime-types';
+import { createTaskQueue, createWorkspace, getAllWorkflows, getAllTaskQueues, createWorkflow, QueueToQueueSid, getAllWorkers } from './task-router-helpers.private';
+import { 
+  WORKSPACE_FRIENDLY_NAME,
+  SCHEDULERS,
+  EDUCATORS,
+  SCHEDULING,
+  EDUCATION,
+  INTAKE_BY_SCHEDULERS,
+  TRANSFER_TO_NURSE_EDUCATOR
+} from './task-router-helpers.private';
+// Fetches specific types
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import { WorkflowInstance } from 'twilio/lib/rest/taskrouter/v1/workspace/workflow';
+
+export const handler: ServerlessFunctionSignature = async function(
+  context: Context,
+  event: {},
+  callback: ServerlessCallback
+) {
+  
+  const client = context.getTwilioClient();
+  const response = new Twilio.Response();
+  response.appendHeader('Access-Control-Allow-Origin', '*');
+
+  try {
+    // Get the Flex Workspace
+    let flexWorkspace = await client.taskrouter.workspaces
+      .list()
+      .then(workspaces => workspaces.find(workspace => WORKSPACE_FRIENDLY_NAME === workspace.friendlyName));
+
+    // If We don't have a flex_workspace, create it.  Flex Accounts only allowed one Workspace.
+    if (!flexWorkspace) flexWorkspace = await createWorkspace(client, WORKSPACE_FRIENDLY_NAME);
```

Comment:
nit: even for single statements, it's still much more readable to utilize your vertical space, especially if you're doing side-effecty code like this.

```tsx
    if (!flexWorkspace) {
         flexWorkspace = await createWorkspace(client, WORKSPACE_FRIENDLY_NAME);
    }
```